# request

---
# response
bool success
string msg